.github/workflows/bors_ci.yml

```
@@ -0,0 +1,711 @@
+# You can find more information about the design decisions here:
+# https://github.com/xaynetwork/xayn_ai/pull/21
+
+name: Bors-Rust-CI
```

```suggestion
name: Merge master CI
```
seems a bit better. And we can rename the other ci something like PR CI or Dev CI.